src/Workspaces/CSharpTest/EmbeddedLanguages/RegularExpressions/CSharpRegexParserTests_TestGeneration.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#if false
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.RegularExpressions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.RegularExpressions
+{
+    internal class LogicalStringComparer : IComparer<string>
+    {
+        [DllImport("shlwapi.dll", CharSet = CharSet.Unicode)]
+        private static extern int StrCmpLogicalW(string psz1, string psz2);
+
+        public static readonly IComparer<string> Instance = new LogicalStringComparer();
+
+        private LogicalStringComparer()
+        {
+        }
+
+        public int Compare(string x, string y)
+        {
+            return StrCmpLogicalW(x, y);
+        }
+    }
+
+    public class Fixture : IDisposable
```

Comment:
>Fixture [](start = 17, length = 7)

Can you explain the overall purpose for this? Why do we need a hard-coded path in this file? (seems wrong to me)

Reply:
Note: this entire file is commented out.  I used it to both quickly generate a huge swath of tests, and to update them efficiently as i decided on new tree-forms or the test output.  If you want i can just kill all of this.  But it was useful during heavy activity